### DISCUSSION TITLE
新版本中的路由

### DISCUSSION THREAD

**Discussion by abooaboo** [yanue/V2rayU#1471] [General]
请问新版本中路由和pac模式的区别在哪里？

**Comment by yjhjerry** [comment]
同问，没有找到相关的文档跟解释...

**Comment by yfan51** [comment]
我下载的v2rayU安装电脑后，怎么是英文版。找不到在哪里更改中文语言。这个怎么办呢？谢谢

**Comment by abooaboo** [comment]
设置里面可以改

Chen Gang


sail ***@***.***>于2024年11月10日 周日20:08写道：
